definitions/npm/ramda_v0.x.x/flow_v0.62.x-/ramda_v0.x.x.js

```diff
@@ -316,35 +316,30 @@ declare module ramda {
     de: UnaryFn<D, E>,
     ef: UnaryFn<E, F>,
     fg: UnaryFn<F, G>,
-    ...rest: Array<void>
-  ) => UnaryFn<A, G>) &
+      ) => UnaryFn<A, G>) &
```

Comment:
Looks like all the indentations here are off... 😬

Reply:
I don't believe I indented them, although it's possible. I noticed there's a pre-commit linter application across anything that's changed. I'll try making the adjustment and see if it's me or the linter.

Reply:
It was me!

I think.